content/docs/user-guide/experiment-management/sharing-experiments.md

```diff
@@ -74,82 +24,134 @@ pulling to/from both Git and DVC remotes in the case of experiments.
-$ dvc remote list
-storage s3://mybucket/my-dvc-store
-```
-
-> ⚠️ Note that DVC can only authenticate with Git remotes using [SSH URLs].
+DVC can only authenticate with Git remotes using [SSH URLs].
 
 [ssh urls]:
   https://git-scm.com/book/en/v2/Git-on-the-Server-The-Protocols#_the_protocols
 
-### Uploading experiments
-
-You can upload an experiment with all of its files and data using
-`dvc exp push`, which takes a Git remote name and an experiment ID or name as
-arguments.
+</admon>
 
-> 💡 You can use `dvc exp show` to find experiment names.
+You can upload an experiment using `dvc exp push --no-cache`, which takes a Git
+remote name and an experiment ID or name. (Remove the `--no-cache` flag to
+include <abbr>cached</abbr> data, which requries a `dvc remote`.) For example:
 
 ```cli
-$ dvc exp push origin quare-zips
+$ dvc exp push --no-cache origin quare-zips
 ```
 
-Once pushed, you can easily [list remote experiments] (with `dvc exp list`).
+<admon type="tip">
+
+Learn how to [list local experiments] (to find their names).
 
+You can also [list remote experiments] (without/before downloading them).
+
+[list local experiments]:
+  /doc/user-guide/experiment-management/comparing-experiments#list-experiments-in-the-project
 [list remote experiments]:
   /doc/user-guide/experiment-management/comparing-experiments#list-experiments-saved-remotely
 
-### Downloading experiments
+</admon>
 
-When you clone a DVC repository, it doesn't fetch any experiments by default. In
-order to get them, use `dvc exp pull` (with the Git remote and the experiment
-name), for example:
+To download an experiment, use `dvc exp pull --no-cache` (with the Git remote
+and experiment name). Remove `--no-cache` to include data, for example:
 
 ```cli
 $ dvc exp pull origin cnn-32
 ```
 
-This pulls all the necessary files from both remotes. Again, you need to have
-both of these configured (see this [earlier section](#preparation)).
-
-If an experiment being pulled already exists in the local project, DVC won't
-overwrite it unless you supply `--force`.
+This puts all the necessary files and data (from both Git and DVC remotes) in
+your project.
 
 ### Sharing many experiments
 
-Use the`--rev` option of `dvc exp push` and `dvc exp pull` to share many
-experiments at once. For example, to upload all experiments based on the latest
-commit of the current branch (Git `HEAD`), use `--rev HEAD`:
+You can use `--rev` command option to push and pull multiple experiments at
+once. For example, to upload all experiments based on the latest commit of the
+current branch (Git `HEAD`), use `--rev HEAD`:
 
 ```
 $ dvc exp push --rev HEAD origin
 ```
+
+### Limitations
+
+- `dvc exp push` is designed to work with a limited set of experiments (a
+  specific one or based on a Git commit).
+- DVC experiments are not fetched when cloning a <abbr>DVC repository</abbr> (to
+  avoid cluttering your local repo). You must `dvc exp pull` the ones you want.
+
+## Method 2: More persistent
```

Comment:
Other ideas:

- More steps, more flexible
- For everyone
- Using a regular Git workflow

Reply:
> For everyone (vs. "developers")

Hmmm so maybe this should be the first method after all 😅